android/build.gradle

```diff
@@ -2,10 +2,11 @@
 
 buildscript {
     repositories {
+        google()// this is the magic line which fixes build failure on yarn android
```

Comment:
Can you please add a better explanation? I don't like the fact that we have to use magic here but we don't know what it actually means.